daily-shoutout/app.py

```diff
@@ -55,11 +56,22 @@ def sendTweet(tweet_string):
     client.create_tweet(text = tweet_string)
 
 
+def returnResponse(response_string):
```

Comment:
Nitpick, but [Python prefers snakecase for function and variable names](https://peps.python.org/pep-0008/#function-and-variable-names):

```suggestion
def return_response(response_string):
```

Better yet, this should actually return the JSON-ified response, right? How about:

```suggestion
def json_response(response_string):
```